packages/core/src/payment/strategies/braintree/braintree-payment-processor.ts

```diff
@@ -38,6 +38,7 @@ export interface PaypalConfig {
     currency: string;
     locale: string;
     offerCredit?: boolean;
+    shippingAddressEditable?: boolean;
```

Comment:
Does this need to be optional?

Reply:
Yes, we can skip adding this option for other strategies